asapdiscovery-dataviz/asapdiscovery/dataviz/tests/test_html_viz.py

```diff
@@ -16,15 +17,43 @@ def test_html_viz_subpockets(pose, protein, target, tmp_path):
     html_visualizer.write_pose_visualizations()
 
 
+@pytest.mark.parametrize("align", [True, False])
+def test_html_viz_subpockets_align(pose, protein, align, tmp_path):
+    html_visualizer = HTMLVisualizer(
+        poses=[pose],
+        output_paths=[tmp_path / "html_viz.html"],
+        target="SARS-CoV-2-Mpro",
+        protein=protein,
+        align=align,
+    )
+    html_visualizer.write_pose_visualizations()
```

Comment:
Do we want to assert something here in the tests? What is the test actually testing?

Reply:
Good point.